src/main/java/org/tarantool/MsgPackLite.java

```diff
@@ -72,13 +72,6 @@
 
     public void pack(Object item, OutputStream os) throws IOException {
         DataOutputStream out = new DataOutputStream(os);
-        if (item instanceof Callable) {
```

Comment:
Remove without workaround? Also your could check the docs here https://docs.oracle.com/javase/7/docs/api/java/util/concurrent/Callable.html

The Callable interface is similar to Runnable, in that both are designed for classes whose instances are POTENTIALLY executed by another thread.